apps/remix-ide-e2e/src/tests/editor.spec.ts

```diff
@@ -37,7 +37,6 @@ module.exports = {
       .sendKeys('*[class="ace_text-input"]', 'error')
       .pause(2000)
       .waitForElementVisible('.ace_error', 60000)
-      .checkAnnotations('error', 28)
```

Comment:
This line runs check for error highlight in editor. It shouldn't be removed.

Reply:
removed it because there is a duplicate. line 43 still check the same thing.

Reply:
bumping this comment, 
Are you sure this need to be removed?
This is tested line 44, yes but the first test still make sense:
line 40: test if the annotation is displayed.
line 42: (we are showing udapp) test if the annotation is removed.
line 44: (back to the compiler) test if the annotation is put back.

line 44 is testing if the annotation is put back but line 40 is testing if the annotation is correctly set in a first place.

Reply:
I see sense in you point and have added it back.  I actually responded to this comment 6 days ago.